src/Component/Data/ExpressionBuilderInterface.php

```diff
@@ -79,11 +79,13 @@ public function greaterThan(string $field, $value);
     public function greaterThanOrEqual(string $field, $value);
 
     /**
+     * @param mixed[] $values
```

Comment:
For `in` and `notIn` at `src/Bundle/Doctrine/DBAL/ExpressionBuilder.php`  `$values` are configured to be array of strings earlier in this PR.